ConsoleApp/Helpers/CoffeeHelper.cs

```diff
@@ -0,0 +1,55 @@
+using Models;
+
+using System.Text.Json;
+
+namespace Helpers
+{
+    public class CoffeeHelper : IDisposable
+    {
+        private readonly string uri = "https://coffee.alexflipnote.dev/random.json";
+        private MyHttpClient? _myHttpClient;
+        private CancellationTokenSource? _cancellationTokenSource;
+
+        public CoffeeHelper()
+        {
+            _myHttpClient = new MyHttpClient();
+        }
+
+        public async Task ActionAsync()
+        {
+            try
+            {
+                _cancellationTokenSource = new(5000);
```

Comment:
it's better to keep `_cancellationTokenSource` locally, in case if you call this method in parallel, one can be no disposed